packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1302,6 +1303,82 @@ export class MergeTree {
 		return undefined;
 	}
 
+	/**
+	 * Finds the nearest reference with ReferenceType.Tile to `startPos` in the direction dictated by `posPrecedesTile`.
+	 * Uses depthFirstNodeWalk in addition to the block-accelerated functionality introduced for findTile. The search
+	 * position will be included in the nodes to walk, so searching on all positions, including the endpoints, can be considered
+	 * inclusive. Any out of bound search positions will return undefined, so in order to search the whole string, a forward
+	 * search can begin at 0, or a bacward search can begin at length-1.This function will replace findTile, which will be deprecated.
+	 *
+	 * @param startPos - Position at which to start the search
+	 * @param clientId - clientId dictating the perspective to search from
+	 * @param tileLabel - Label of the tile to search for
+	 * @param tilePrecedesPos - Whether the desired tile comes before (true) or after (false) `startPos`
```

Comment:
@Abe27342 - I think we should keep this variable name as is, based on how Word uses the value. They have a "preceding" variable, where the intent is to search forward when the value is false, and search backward when the value is true. I kept the leftmost/rightmost logic in tileShift the same as well, but walkToFindTile uses the correct search direction, where findTile did not.

Reply:
tilePrecedesPos is awkward though and requires thinking to parse what it means. I'm guessing that contributed to it being accidentally reversed. Can't we just use something obvious like forward/backward?